src/state/slices/tradeQuoteSlice/tradeQuoteSlice.ts

```diff
@@ -130,6 +131,7 @@ export const tradeQuoteSlice = createSlice({
       } else {
         // complete the second hop
         state.tradeExecution.secondHop.swap.state = TransactionExecutionState.Complete
+        state.tradeExecution.secondHop.swap.message = undefined
```

Comment:
clearing the execution message on complete to avoid showing messages like "awaiting XYZ" etc after a trade is completed.